README.md

```diff
@@ -9,36 +9,50 @@ Design documentation is currently in a
 Centos 6
 Centos 7?
 
-## Attributes
+## Running tests
 
-<table>
-  <tr>
-    <th>Key</th>
-    <th>Type</th>
-    <th>Description</th>
-    <th>Default</th>
-  </tr>
-  <tr>
-    <td><tt>['create']['bacon']</tt></td>
-    <td>Boolean</td>
-    <td>whether to include bacon</td>
-    <td><tt>true</tt></td>
-  </tr>
-</table>
+To run all tests, including style checks with both foodcritic and rubocop,
+we use Rake. Rake allows for a granular level of testing, including running
+integration, style, and unit testing from one tool.
 
-## Usage
+To run all tests, if you've set up your cloud environment, run:
 
-### create::default
+```
+$ rake cloud
+```
+
+Otherwise, you can run all tests using a Vagrant virtual machine with:
+
+```
+$ rake
```

Comment:
Perhaps this should come first, followed by "If you have access to an openstack instance (etc.) you can use `rake cloud`"